--- curl-7.49.1/docs/Makefile.am.0006~	2016-05-27 16:09:12.000000000 +0200
+++ curl-7.49.1/docs/Makefile.am	2016-05-31 16:23:52.940776715 +0200
@@ -29,7 +29,7 @@ PDFPAGES = curl.pdf curl-config.pdf mk-c
 
 HTMLPAGES = $(GENHTMLPAGES) index.html
 
-SUBDIRS = examples libcurl
+SUBDIRS = libcurl
 
 CLEANFILES = $(GENHTMLPAGES) $(PDFPAGES)
 
--- curl-7.49.1/docs/Makefile.in.0006~	2016-05-31 16:23:52.941776710 +0200
+++ curl-7.49.1/docs/Makefile.in	2016-05-31 16:24:48.193510284 +0200
@@ -446,7 +446,7 @@ noinst_man_MANS = mk-ca-bundle.1
 GENHTMLPAGES = curl.html curl-config.html mk-ca-bundle.html
 PDFPAGES = curl.pdf curl-config.pdf mk-ca-bundle.pdf
 HTMLPAGES = $(GENHTMLPAGES) index.html
-SUBDIRS = examples libcurl
+SUBDIRS = libcurl
 CLEANFILES = $(GENHTMLPAGES) $(PDFPAGES)
 EXTRA_DIST = MANUAL BUGS CONTRIBUTE FAQ FEATURES INTERNALS SSLCERTS     \
  README.win32 RESOURCES TODO TheArtOfHttpScripting THANKS VERSIONS      \
