--- curl-7.30.0/curl-config.in.0004~	2013-04-18 12:26:27.355994049 +0200
+++ curl-7.30.0/curl-config.in	2013-04-18 12:27:42.583053581 +0200
@@ -142,16 +142,7 @@ while test $# -gt 0; do
         ;;
 
     --libs)
-        if test "X@libdir@" != "X/usr/lib" -a "X@libdir@" != "X/usr/lib64"; then
-           CURLLIBDIR="-L@libdir@ "
-        else
-           CURLLIBDIR=""
-        fi
-        if test "X@REQUIRE_LIB_DEPS@" = "Xyes"; then
-          echo ${CURLLIBDIR}-lcurl @LIBCURL_LIBS@
-        else
-          echo ${CURLLIBDIR}-lcurl
-        fi
+        pkg-config --libs libcurl
         ;;
 
     --static-libs)
