diff -up curl-8.20.0/curl-config.in.4~ curl-8.20.0/curl-config.in
--- curl-8.20.0/curl-config.in.4~	2026-04-29 07:45:57.000000000 +0200
+++ curl-8.20.0/curl-config.in	2026-05-23 18:04:39.332825633 +0200
@@ -74,7 +74,7 @@ while test "$#" -gt 0; do
     ;;
 
   --cc)
-    echo '@CC@'
+    echo 'cc'
     ;;
 
   --prefix)
@@ -149,16 +149,7 @@ while test "$#" -gt 0; do
     ;;
 
   --libs)
-    if test "@libdir@" != '/usr/lib' && test "@libdir@" != '/usr/lib64'; then
-      curllibdir="-L@libdir@ "
-    else
-      curllibdir=''
-    fi
-    if test '@ENABLE_SHARED@' = 'no'; then
-      echo "${curllibdir}-lcurl @LIBCURL_PC_LIBS_PRIVATE@"
-    else
-      echo "${curllibdir}-lcurl"
-    fi
+    pkg-config libcurl --libs
     ;;
 
   --ssl-backends)
