diff -uraN curl-8.10.1/curl-config.in omv-curl-8.10.1/curl-config.in
--- curl-8.10.1/curl-config.in	2024-09-18 07:56:40.000000000 +0200
+++ omv-curl-8.10.1/curl-config.in	2024-09-19 17:53:02.090042241 +0200
@@ -75,7 +75,7 @@
     ;;
 
   --cc)
-    echo '@CC@'
+    echo 'cc'
     ;;
 
   --prefix)
@@ -155,16 +155,7 @@
     ;;
 
   --libs)
-    if test "X@libdir@" != "X/usr/lib" -a "X@libdir@" != "X/usr/lib64"; then
-      CURLLIBDIR="-L@libdir@ "
-    else
-      CURLLIBDIR=""
-    fi
-    if test "X@ENABLE_SHARED@" = "Xno"; then
-      echo "${CURLLIBDIR}-lcurl @LIBCURL_PC_LIBS_PRIVATE@"
-    else
-      echo "${CURLLIBDIR}-lcurl"
-    fi
+    pkg-config libcurl --libs
     ;;
 
   --ssl-backends)
