diff -up curl-8.8.0/curl-config.in.3~ curl-8.8.0/curl-config.in
--- curl-8.8.0/curl-config.in.3~	2024-06-03 15:42:32.484151492 +0200
+++ curl-8.8.0/curl-config.in	2024-06-03 15:44:56.618133015 +0200
@@ -73,7 +73,7 @@ while test "$#" -gt 0; do
     ;;
 
   --cc)
-    echo '@CC@'
+    echo 'cc'
     ;;
 
   --prefix)
@@ -153,16 +153,7 @@ while test "$#" -gt 0; do
     ;;
 
   --libs)
-    if test "X@libdir@" != "X/usr/lib" -a "X@libdir@" != "X/usr/lib64"; then
-      CURLLIBDIR="-L@libdir@ "
-    else
-      CURLLIBDIR=""
-    fi
-    if test "X@ENABLE_SHARED@" = "Xno"; then
-      echo "${CURLLIBDIR}-lcurl @LIBCURL_LIBS@"
-    else
-      echo "${CURLLIBDIR}-lcurl"
-    fi
+    pkg-config libcurl --libs
     ;;
 
   --ssl-backends)
