--- curl-7.58.0/curl-config.in.0004~	2018-01-24 09:24:50.180891348 +0100
+++ curl-7.58.0/curl-config.in	2018-01-24 09:26:16.846340020 +0100
@@ -143,16 +143,7 @@ while test $# -gt 0; do
         ;;
 
     --libs)
-        if test "X@libdir@" != "X/usr/lib" -a "X@libdir@" != "X/usr/lib64"; then
-           CURLLIBDIR="-L@libdir@ "
-        else
-           CURLLIBDIR=""
-        fi
-        if test "X@REQUIRE_LIB_DEPS@" = "Xyes"; then
-          echo ${CURLLIBDIR}-lcurl @LIBCURL_LIBS@
-        else
-          echo ${CURLLIBDIR}-lcurl
-        fi
+        pkg-config --libs libcurl
         ;;
     --ssl-backends)
         echo "@SSL_BACKENDS@"
