diff -up curl-8.20.0/m4/curl-mbedtls.m4.3~ curl-8.20.0/m4/curl-mbedtls.m4
--- curl-8.20.0/m4/curl-mbedtls.m4.3~	2026-04-29 07:45:57.000000000 +0200
+++ curl-8.20.0/m4/curl-mbedtls.m4	2026-05-23 18:03:23.797762526 +0200
@@ -60,12 +60,12 @@ if test "x$OPT_MBEDTLS" != "xno"; then
 
     if test "$USE_MBEDTLS" != "yes"; then
       dnl add the path and test again
-      addld=-L$OPT_MBEDTLS/lib$libsuff
-      addcflags=-I$OPT_MBEDTLS/include
+      #addld=-L$OPT_MBEDTLS/lib$libsuff
+      #addcflags=-I$OPT_MBEDTLS/include
       mbedtlslib=$OPT_MBEDTLS/lib$libsuff
 
-      LDFLAGS="$LDFLAGS $addld"
-      LDFLAGSPC="$LDFLAGSPC $addld"
+      #LDFLAGS="$LDFLAGS $addld"
+      #LDFLAGSPC="$LDFLAGSPC $addld"
       if test "$addcflags" != "-I/usr/include"; then
         CPPFLAGS="$CPPFLAGS $addcflags"
       fi
