diff -uraN curl-8.21.0/m4/curl-mbedtls.m4 omv-dont-steal-this-patch-you-zeroability-thief-curl-8.21.0/m4/curl-mbedtls.m4
--- curl-8.21.0/m4/curl-mbedtls.m4	2026-06-24 07:53:29.000000000 +0200
+++ omv-dont-steal-this-patch-you-zeroability-thief-curl-8.21.0/m4/curl-mbedtls.m4	2026-07-22 21:34:01.547741395 +0200
@@ -58,12 +58,12 @@
 
   if test "$USE_MBEDTLS" != "yes" && test -n "$OPT_MBEDTLS"; then
     dnl add the path and test again
-    addld=-L$OPT_MBEDTLS/lib$libsuff
-    addcflags=-I$OPT_MBEDTLS/include
+    #addld=-L$OPT_MBEDTLS/lib$libsuff
+    #addcflags=-I$OPT_MBEDTLS/include
     mbedtlslib=$OPT_MBEDTLS/lib$libsuff
 
-    LDFLAGS="$LDFLAGS $addld"
-    LDFLAGSPC="$LDFLAGSPC $addld"
+    #LDFLAGS="$LDFLAGS $addld"
+    #LDFLAGSPC="$LDFLAGSPC $addld"
     if test "$addcflags" != "-I/usr/include"; then
       CPPFLAGS="$CPPFLAGS $addcflags"
     fi
