diff -up curl-8.22.0/configure.ac.omv~ curl-8.22.0/configure.ac
--- curl-8.22.0/configure.ac.omv~	2026-09-02 00:00:00.000000000 +0000
+++ curl-8.22.0/configure.ac	2026-09-02 00:00:00.000000000 +0000
@@ -2814,7 +2814,9 @@ AS_HELP_STRING([--disable-versioned-symb
   *)
     AC_MSG_RESULT(yes)
     AC_MSG_CHECKING([if libraries can be versioned])
-    GLD=`$LD --help < /dev/null 2>/dev/null | $GREP version-script`
+    TEST_LD="$LD"
+    if test -z "$TEST_LD"; then TEST_LD="ld"; fi
+    GLD=`$TEST_LD --help < /dev/null 2>/dev/null | $GREP version-script`
     if test -z "$GLD"; then
       AC_MSG_RESULT(no)
       AC_MSG_WARN([You need an ld version supporting the --version-script option])
