diff -uraN curl-8.10.1/m4/curl-mbedtls.m4 omv-curl-8.10.1/m4/curl-mbedtls.m4
--- curl-8.10.1/m4/curl-mbedtls.m4	2024-09-18 07:56:40.000000000 +0200
+++ omv-curl-8.10.1/m4/curl-mbedtls.m4	2024-09-19 17:39:27.097918096 +0200
@@ -60,11 +60,11 @@
 
     if test "x$USE_MBEDTLS" != "xyes"; then
       dnl add the path and test again
-      addld=-L$OPT_MBEDTLS/lib$libsuff
-      addcflags=-I$OPT_MBEDTLS/include
+      #addld=-L$OPT_MBEDTLS/lib$libsuff
+      #addcflags=-I$OPT_MBEDTLS/include
       mbedtlslib=$OPT_MBEDTLS/lib$libsuff
 
-      LDFLAGS="$LDFLAGS $addld"
+      #LDFLAGS="$LDFLAGS $addld"
       if test "$addcflags" != "-I/usr/include"; then
         CPPFLAGS="$CPPFLAGS $addcflags"
       fi
