diff -Naur curl-7.79.1/configure.ac curl-7.79.1.tpg/configure.ac
--- curl-7.79.1/configure.ac	2021-11-04 09:24:40.955173710 +0000
+++ curl-7.79.1.tpg/configure.ac	2021-11-04 09:25:55.974844369 +0000
@@ -372,10 +372,6 @@
 dnl **********************************************************************
 
 CURL_CHECK_COMPILER
-CURL_SET_COMPILER_BASIC_OPTS
-CURL_SET_COMPILER_DEBUG_OPTS
-CURL_SET_COMPILER_OPTIMIZE_OPTS
-CURL_SET_COMPILER_WARNING_OPTS
 
 if test "$compiler_id" = "INTEL_UNIX_C"; then
   #
